vaadin-map-flow-parent/vaadin-map-flow-integration-tests/src/main/java/com/vaadin/flow/component/map/demo/SamplerView.java

```diff
@@ -1,8 +1,9 @@
-package com.vaadin.flow.component.map;
+package com.vaadin.flow.component.map.demo;
```

Comment:
Moved this view to demos, and renamed to `SamplerView` as it was only used as a manual testing playground, and is not used in an IT.